### PR TITLE
[#1] [Chore] Setup project - Part 2 (re-merge)

### LINKED ISSUE #1: Set up project repository
## Why

When starting a new project, the project repository foundations need to be in placed for smooth application's development with proper app building and running tests guidance.

## Acceptance Criteria
- [ ] Add pull request and issue templates.
- [ ] Setup proper `.gitignore` file for the project.
- [ ] Setup all necessary 3rd party libraries such as: RxSwift, Alamofire, etc. and library management tools such as: CocoaPods, Swift Package Manager, etc.
- [ ] Run the script from iOS template to generate initial classes and folder structure or create manually.
- [ ] Setup `README.md` file with proper description, setup/testing instruction, features requirements and API specs links.

### PR DESCRIPTION
**Pull request by markgravity**
Resolved #1 

## What happened

When starting a new project, the project repository foundations need to be in place for smooth application development with proper app building and running tests guidance.

## Insight

- Create xcode project and folder structure
- Add staging & production schema
 
## Proof Of Work
Folder Structure
Screen Shot 2021-08-06 at 09 33 35

Staging Build

Production Build

### CODE REVIEW

File: NimbleMedium.xcodeproj/xcshareddata/xcschemes/NimbleMedium (production).xcscheme [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (suho) [inline NimbleMedium.xcodeproj/xcshareddata/xcschemes/NimbleMedium (production).xcscheme:1]: Please follow the naming from our documentation: https://github.com/nimblehq/ios-templates/wiki/Project-Configurations#schemes
It should be `NimbleMedium`

File: NimbleMedium.xcodeproj/xcshareddata/xcschemes/NimbleMedium (staging).xcscheme [hunk 2]
```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (suho) [inline NimbleMedium.xcodeproj/xcshareddata/xcschemes/NimbleMedium (staging).xcscheme:1]: Please follow our documentation to name de scheme: https://github.com/nimblehq/ios-templates/wiki/Project-Configurations#schemes

File: NimbleMedium.xcodeproj/xcshareddata/xcschemes/NimbleMedium (production).xcscheme (hunk 1 above)
Comment (markgravity) [reply NimbleMedium.xcodeproj/xcshareddata/xcschemes/NimbleMedium (production).xcscheme]: @suho I updated 🙏

File: NimbleMedium.xcodeproj/xcshareddata/xcschemes/NimbleMedium (staging).xcscheme (hunk 2 above)
Comment (markgravity) [reply NimbleMedium.xcodeproj/xcshareddata/xcschemes/NimbleMedium (staging).xcscheme]: @suho I updated 🙏

**Review by minhnimble — APPROVED** [review]
LGTM 🚀

**Review by suho — APPROVED** [review]

**Pull request merged**